dotnet-desktop-guide/net/wpf/events/how-to-add-an-event-handler-using-code.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "How to add an event handler using code"
+description: Learn how to add an event handler in code-behind for an element in Windows Presentation Foundation (WPF).
+ms.date: "02/01/2021"
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords:
+  - "event handlers [WPF], adding"
+  - "XAML [WPF], adding event handlers"
+---
+<!-- The acrolinx score was 99 on 02/01/2021-->
```

Comment:
I'm sorry but this should be 100.... just kidding, good job!

Reply:
@adegeo thanks for the review :)